bundles/org.openhab.ui/web/src/assets/addon-store.js

```diff
@@ -41,6 +41,27 @@ export const AddonSuggestionLabels = {
   }
 }
 
+export const AddonConnectionTypes = {
+  cloud: {
+    label: 'Cloud allowed',
+    values: ['none', 'local', 'hybrid', 'cloud']
+  },
+  cloudConfig: {
+    label: 'Cloud configuration only',
```

Comment:
This text is slightly misleading, since according to the current [definition](https://www.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions) hybrid means "add-ons that interact locally without internet access, but can optionally use a cloud connection for extended functionality (such as discovery)". This is not specific to configuration.

Reply:
Would ‘cloud discovery only’ fit better? I need to keep it short to fit the UI on a mobile.

Reply:
I would also reconsider "Exclude not for country" as it's quite puzzling IMO.

Suggestion:
- All applicable to you
- Only your country
- All

Reply:
> Would ‘cloud discovery only’ fit better? I need to keep it short to fit the UI on a mobile.

I don't think so, since discovery is just an example. You could have a binding where 20% of the (optional) functionality is provided through a cloud service. Unfortunately I don't have a good suggestion for a short way to put this. Maybe something like "Optional cloud functionality"?